### DISCUSSION TITLE
[archived] "Running at Massive Scale" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10146] [General]
# argo-workflows/running-at-massive-scale/

https://argoproj.github.io/argo-workflows/running-at-massive-scale/

**Comment by caelan-io** [comment]
Also, upgrade to K8s 1.27+ if possible to take advantage of more K8s API server capacity:
https://pipekit.io/blog/upgrade-to-kubernetes-127-take-advantage-of-performance-improvements
